github/spec/features/configure_repos_spec.rb

```diff
@@ -107,7 +107,7 @@
   end
 
   def given_theres_a_repo(archived: false,
-                          full_name: "alphagov/publishing-api",
+                          full_name: "alphagov/smart-sandwich",
```

Comment:
```
                          full_name: "alphagov/some-repo-name",
```

...might be better, if less fun :)

Reply:
Oh, we're definitely sticking with "smart-sandwich". It's a critical part of our infra.